snuba/migrations/snuba_migrations/metrics/0024_metrics_distributions_add_histogram.py

```diff
@@ -0,0 +1,81 @@
+from typing import Sequence
+
+from snuba.clickhouse.columns import AggregateFunction, Column, Float
+from snuba.clusters.storage_sets import StorageSetKey
+from snuba.migrations import migration, operations
+from snuba.migrations.snuba_migrations.metrics.templates import (
+    COL_SCHEMA_DISTRIBUTIONS_V2,
+    get_forward_view_migration_polymorphic_table_v2,
+    get_polymorphic_mv_v3_name,
+)
+
+
+class Migration(migration.ClickhouseNodeMigration):
+    """
+    Add HdrHistogram to distribution metrics.
+    """
+
+    blocking = False
+    raw_table_name = "metrics_raw_local"
```

Comment:
I see that in migration 0023 we are referencing metrics_raw_v2_local. Is that the new one I should use?

Reply:
yes, it is